### PR TITLE
Type

### PR DESCRIPTION
**Pull request by alxmrtnz**
These changes update typography styles.

The type sizing and rhythm is a slightly modified 'augmented fourth' type scale:
screen shot 2016-04-08 at 4 18 49 pm

Attached is a screenshot documenting the before and after. 
type-style-update-48-16

Connects to #34

### CODE REVIEW

File: styles/components/_typography.scss [hunk 1]
```diff
@@ -6,26 +6,28 @@
 %heading--jumbo,
 .heading--jumbo {
```
Comment (heyitsolivia) [inline styles/components/_typography.scss]: @alxmrtnz If we're no longer using `.heading--jumbo` anywhere in the templates, we can just remove this entire class.

**Comment by heyitsolivia** [comment]
@alxmrtnz Looks like you duplicated some files into the root directory. Once you remove those and make the change above, I can give a thumb and pull :)

**Comment by alxmrtnz** [comment]
@heyitsolivia ready for thumbs and pull

**Comment by heyitsolivia** [comment]
:+1:

**Pull request merged**